core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/Group.java

```diff
@@ -42,16 +38,9 @@ public Group(String groupID, String groupName) {
     }
 
     public Group(String groupID, String groupName, String displayName, String tenantDomain, String userStoreDomain,
```

Comment:
Let's add doc comments constructors and public methods

Reply:
Updated the PR with the change.